### PR TITLE
SAK-30079 conversion for auto_forward column from boolean to integer

### PR DESCRIPTION
**Pull request by plukasew**
https://jira.sakaiproject.org/browse/SAK-30079

Missing conversion script for SAK-30079

### CODE REVIEW

File: docs/conversion/sakai_21_mysql_conversion.sql [hunk 1]
```diff
@@ -4,3 +4,6 @@ DELETE SAKAI_MESSAGE_BUNDLE from SAKAI_MESSAGE_BUNDLE where PROP_VALUE is NULL;
 -- this constraint may have been missed, it is ok if this line fails just comment it out
 ALTER TABLE CONTENTREVIEW_ITEM ADD CONSTRAINT UK_8dngr1v68kkv4u11c1nvrjj1l UNIQUE (PROVIDERID, CONTENTID);
 
+-- SAK-30079
+ALTER TABLE MFR_PRIVATE_FORUM_T MODIFY COLUMN AUTO_FORWARD INT NOT NULL DEFAULT 2;
```
Comment (ottenhoff) [inline docs/conversion/sakai_21_mysql_conversion.sql:8]: how about TINYINT(1)

Comment (plukasew) [reply docs/conversion/sakai_21_mysql_conversion.sql:8]: AutoDDL creates it as INT apparently, wanted to keep it consistent. I agree it is wasteful when the possible values are only 0-2. I can change it if you want.

Comment (ottenhoff) [reply docs/conversion/sakai_21_mysql_conversion.sql:8]: i dont think it matters much but tinyint(1) is how it was created in the past 

msgcntr/messageforums-app/src/sql/mysql/sakai_messageForums.sql

Comment (plukasew) [reply docs/conversion/sakai_21_mysql_conversion.sql:8]: Yeah I saw that but I think that is from back when it was a boolean value and MySQL didn't have the BIT column type.

Comment (plukasew) [reply docs/conversion/sakai_21_mysql_conversion.sql:8]: I think we could possibly run into JDBC issues with tinyint(1) here as it appears that by default Connector/J treats tinyint(1) like bit.

https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html

Comment (ottenhoff) [reply docs/conversion/sakai_21_mysql_conversion.sql:8]: okay sounds like a future nightmare to unwind. i'm sure we have tons of tinyints all over the place

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**